layouts/_default/list.html

```diff
@@ -1,24 +1,3 @@
 {{ define "main" }}
-<main>
-  <article>
-    <header>
-      <h1>{{.Title}}</h1>
-    </header>
 
```

Comment:
Is this page meant to only contain an empty `define main`?

Reply:
I meant to delete. WIll do.